File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/3_String.js
public class StringInJava {
    public static void main(String[] args) {
    String str = "hello word"; //NOTE: String ko the thay doi dc cac p.tu ben trong no
    String res = "";

    for (int i=0; i < str.length(); i++){
        if(i%2 == 0){
            res += "_";
        }else{
            res += str.charAt(i);
        }
    }
    System.out.println("result: " + res);
    }
}

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/PYTHON/3_String.py
if __name__ == '__main__':
    str = "hello word" #NOTE: String ko the thay doi dc cac p.tu ben trong no
    res = ''

    for i in range(len(str)):
        if i%2 == 0:
            res += '_'
        else:
            res += str[i]

    print('result: '+res)